libcontainer/intelrdt/monitoring_test.go

```diff
@@ -54,7 +54,7 @@ func mockResctrlL3_MON(NUMANodes []string, mocks map[string]uint64) (string, err
 		}
 
 		for fileName, value := range mocks {
-			err := ioutil.WriteFile(filepath.Join(numaPath, fileName), []byte(strconv.FormatUint(value, 10)), 777)
+			err := ioutil.WriteFile(filepath.Join(numaPath, fileName), []byte(strconv.FormatUint(value, 10)), 0777)
```

Comment:
Just for the sake of sanity, I'd prefer `0644` here. Or, using the latest and greatest golang, `0o644`.

Reply:
I wanted to respect the original commit in this pull request.
But if maintainers don't mind it, I'll change from `0777` to `0o644` in this pull request.

Reply:
@kolyshkin I changed the file mode. Thanks!